packages/yarnpkg-core/sources/StreamReport.ts

```diff
@@ -126,7 +127,7 @@ export class StreamReport extends Report {
         ? `${name}: ${message}`
         : message;
 
-      report.reportWarning(MessageName.UNNAMED, fullMessage);
+      report.writeReportWarning(MessageName.UNNAMED, fullMessage);
```

Comment:
Can't we just keep calling `writeReportInfo`, and mark this as a private function? It adds a small cognitive overhead I'd prefer to avoid.

Reply:
Oh yeah, this should still be `reportWarning` I guess 🤔